### PR TITLE
Redesign top navbar

### PR DESCRIPTION
**Pull request by Splines**
**Redesign of the top navbar**.
⚠ This PR should be merged into `mampf-next` (not `feature/left-navbar-design`) _after_ having merged #446 into `mampf-next`.

## Features

- Add background image to navbar (responsive)
- Add shadow to navbar
- Use SVG version of MaMpf logo

## Preview
image

### CODE REVIEW

**Review by christian-heusel — CHANGES_REQUESTED** [review]
In my opinion the Image background for the top navbar is not actually an improvement over the solid one. 
Sorry, but currently thats a NACK from me.

**Comment by Splines** [comment]
@christian-heusel True, it might even distract with the image. So, what do you think of this version:
image

**Review by christian-heusel — APPROVED** [review]
I think that looks better, so I approve the changes!
I dont think I can judge this PR from a technical standpoint 🤷🏻 

Edit: but since @fosterfarrell9 is also a reviewer I guess he will have some more input wrt implementation ;)

**Comment by Splines** [comment]
Current look

image

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -309,4 +313,13 @@ def get_class_for_any_path(paths)
...
 
+  def get_class_for_any_path_startswith(paths)
+    for path in paths do
+      if request.path.starts_with?(path)
+        return $active_css_class
+      end
+    end
+    return ''
+  end
+
```
Comment (fosterfarrell9) [inline app/helpers/application_helper.rb:324]: I would suggest refactoring this into something more rubyesque like
```
def get_class_for_any_path_startswith(paths)
  if paths.any? { |path| request.path.starts_with?(path) }
    return $active_css_class
  end
  return ''
end
```

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -309,4 +313,13 @@ def get_class_for_any_path(paths)
     return ''
   end
```
Comment (fosterfarrell9) [inline app/helpers/application_helper.rb:315]: For a possible refactorisation of this method see PR #446.

**Review by fosterfarrell9 — APPROVED** [review]

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (Splines) [reply app/helpers/application_helper.rb:317]: Yeah, this is similar to Python list comprehension, didn't know Ruby had that ;)

File: app/assets/stylesheets/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            position: absolute;
+            top: 21px;
+            left: 16px;
+        }
+    }
+}
+
+.new-comment {
+    color: #ffc3d0 !important;
+}
```
[the remark this answers is not in the document]
Comment (Splines) [reply app/assets/stylesheets/navbar.scss:43]: looks good to me on the blue background 👍

**Pull request merged**